tests/validation/validate_kinetic.hpp

```diff
@@ -23,11 +28,11 @@ void run_kinetic_dt(nest::mc::cell& c, float t_end, nlohmann::json meta, const s
     };
 
     meta["sim"] = "nestmc";
-    meta["backend"] = LoweredCell::backend::name();
+    meta["backend"] = backend==backend_policy::use_multicore? "cpu" : "gpu"; // FIXME: fragile
```

Comment:
Yup, it is. We should either just change the key to `backend_policy` and defer the problem, or add an interface to `model` that allows querying the name of an implementation corresponding to a gid.